src/login/pam_systemd.c

```diff
@@ -366,6 +368,14 @@ _public_ PAM_EXTERN int pam_sm_open_session(
 
         remote = !isempty(remote_host) && !is_localhost(remote_host);
 
+        memory_max = pam_getenv(handle, "SESSION_LIMIT_MEMORY_MAX");
+        tasks_max = pam_getenv(handle, "SESSION_LIMIT_TASKS_MAX");
+        cpu_weight = pam_getenv(handle, "SESSION_LIMIT_CPU_WEIGHT");
+        startup_cpu_weight = pam_getenv(handle, "SESSION_LIMIT_STARTUP_CPU_WEIGHT");
```

Comment:
the startup stuff doesn't make too much sense I figure... user sessions don't run in the start-up phase of the system, hence the props have no effect